tests/environment-setup/requirements/requirements.in

```diff
@@ -5,3 +5,5 @@ pytest-aiohttp
 pytest-instafail
 pytest-runner
 pytest-sugar
+
+pyyaml

```

Comment:
Was it correct to add this requirement here?

Reply:
yes, I added a test that required that library